doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -516,6 +516,14 @@ Thread samples
 
 * Removed support for the ``nrf5340dk/nrf5340/cpuapp/ns`` build target for all samples.
 
+* :ref:`ot_cli_sample` sample:
```

Comment:
Remove the changelog entries from this PR and add them as a suggestion in my https://github.com/nrfconnect/sdk-nrf/pull/19240.

Reply:
removed and added suggestion